doc/guides/encoding/rle_lossless.rst

```diff
@@ -0,0 +1,163 @@
+|            +-----------------+-----------------+            |         |
+|            | PALETTE COLOR   | 0               |            |         |
++------------+-----------------+-----------------+------------+---------+
+| 3          | RGB             | 0               | 8 or 16    | 1 to 16 |
+|            +-----------------+                 +------------+---------+
+|            | YBR_FULL        |                 | 8          | 1 to 8  |
++------------+-----------------+-----------------+------------+---------+
+
+To ensure you have the correct *Photometric Interpretation* when encoding using
+*RLE Lossless*, the uncompressed pixel data should already be in the
+corresponding color space:
+
+* If your uncompressed pixel data is grayscale (intensity) based:
+
+  * Use ``MONOCHROME1`` if the minimum intensity value should be displayed as
+    white.
+  * Use ``MONOCHROME2`` if the minimum intensity value should be displayed as
+    black.
+
+* If your uncompressed pixel data uses a single sample per pixel and is an index
+  to the :dcm:`Red, Green and Blue Palette Color Lookup Tables
+  <part03/sect_C.7.6.3.html#sect_C.7.6.3.1.5>`:
+
+  * Use ``PALETTE COLOR``.
+
+* If your uncompressed pixel data is in RGB color space:
+
+  * For *Photometric Interpretation* ``RGB`` nothing else is required.
+  * For *Photometric Interpretation* ``YBR_FULL``
+
+    * For *Bits Allocated* and *Bits Stored* less than or equal to 8: pixel
+      data must be :func:`converted into YCbCr color space
+      <pydicom.pixel_data_handlers.convert_color_space>`. However
+      you should keep in mind that the conversion operation is lossy.
+    * For *Bits Allocated* and *Bits Stored* between 9 and 16 (inclusive):
+      pixel data should be downscaled to 8-bit (with *Bits Stored*, *Bits
+      Allocated* and *High Bit* updated accordingly) and converted to `YCbCr
+      <https://en.wikipedia.org/wiki/YCbCr>`_ color space. Both of these
+      operations are lossy.
+
+* If your uncompressed pixel data is in `YCbCr
+  <https://en.wikipedia.org/wiki/YCbCr>`_ color space:
+
+  * For *Photometric Interpretation* ``RGB`` the pixel data must first be
+    :func:`converted into RGB color space
+    <pydicom.pixel_data_handlers.convert_color_space>`. However the conversion
+    operation is lossy.
+  * For *Photometric Interpretation* ``YBR_FULL`` nothing else is required.
+
+If a change is made to existing *Pixel Data*, such as conversion to a different
+color space or downsampling to 8-bit then a new *SOP Instance UID* should be
+generated.
+
+You might be asking why you would convert uncompressed RGB pixel data to YCbCr
+(or vice versa) if the conversion itself is lossy. The answer is that
+using YCbCr data should result in a higher compression ratio than
+with RGB, while YCbCr data is usually converted back to RGB before viewing.
+The decision to change the color space should be made with the intended
+usage of your dataset in mind.
```

Comment:
Maybe add a caution that for some uses (like using MG images for diagnostic purposes) it is explicitely forbidden to use lossy compression.

Reply:
Do you have a reference for this? I looked in the standard but couldn't see anything.

Reply:
Or are you referring to the requirements of health regulators?

Reply:
You are right, these are not in the DICOM standard, and I have to look up myself where the requirement comes from. I take it back, mentioning the intended usage as you do shall be sufficient.

Reply:
I did find this:

> The context where the usage of lossy compression of medical images is clinically acceptable is beyond the scope of the DICOM Standard.